resources/js/processes-catalogue/components/ProcessesCatalogue.vue

```diff
@@ -141,7 +143,7 @@ export default {
           .then((response) => {
             this.listCategories = [...this.listCategories, ...response.data.data];
             this.totalPages = response.data.meta.total_pages !== 0 ? response.data.meta.total_pages : 1;
-
+            this.categoryCount = this.listCategories.length;
```

Comment:
Use data.meta.total

Reply:
Please use the permission in order to see the new option, this is not necesary

Reply:
solved